src/validation/error.messages.ts

```diff
@@ -58,8 +58,8 @@ export enum ErrorMessages {
   CITY_OR_TOWN_LEGAL_ENTITY_BO = "Enter the city or town",
   COUNTY = "Enter a county",
   COUNTRY = "Select a country from the list",
-  COUNTRY_INDIVIDUAL_BO = "Start typing, then select a country from the list",
-  COUNTRY_LEGAL_ENTITY_BO = "Start typing, then select a country from the list",
+  COUNTRY_INDIVIDUAL_BO = "Select a country from the list",
```

Comment:
As text is the same and it's for a similar thing (a BO), there's an opportunity to reduce the LOC and have just one constant - e.g. `COUNTRY_BO`. Up to you.